### ISSUE TITLE
TCL "Lab Residential Zone" has no balcony in scientists' quarters

### ISSUE THREAD

**Issue by toweroglass** [CleverRaven/Cataclysm-DDA#73980]
### Describe the bug

I noticed some skitterbots "walking on thin air" in the scientist's residential zone. they are dead in the save i attached. took me a second to realize they were walking on solid ground.

I would assume this should be open air just like the security residential area is. it has a similar layout.

### Attach save file

[Parole-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/15384297/Parole-trimmed.tar.gz)

### Steps to reproduce

see screenshot below. go to tcl residential zone. walk off balcony. don't fall. 
If this is intentional i think it's counterintuitive seeing as these areas all look very similar and you would expect this to be open air.

### Expected behavior

i expected there to be open air where there is not.

### Screenshots

![residential zone 2nd floor](https://github.com/CleverRaven/Cataclysm-DDA/assets/81603517/cb179046-68a1-43e9-bb65-4695f38d8409)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4412 (22H2)
- Game Version: cdda-experimental-2024-05-17-1356 03d8ffb [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Slowdown Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Alternative Map Key [alt_map_key],
    Translate Complex Dialogue [translate_dialogue],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro],
    Mind Over Matter [mindovermatter],
    Extra Mutated Scenarios [extra_mut_scens]
]

**Comment by toweroglass** [comment]
Hopefully this is sufficient. it's super late and i should be going to bed anyway but this seemed like an easy thing to report.

I've been in this area countless times and it just **now** occurred to me that it seems inconsistent.

**Comment by Procyonae** [comment]
Yup I don't remember that being a thing before but those maps use 3 palettes so one of them probably overrides the intended  open air
